services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -447,6 +450,56 @@ def _by_type_name(ec2: EC2InstanceTypeGet) -> bool:
         raise ClustersKeeperNotAvailableError from exc
 
 
+async def _check_project_node_has_all_required_inputs(
```

Comment:
Q: would it be possible (in a different PR is OK) to move away this code. I would like to split at some point `projects_api` which is already reaching 2000 lines!